pkg/services/authn/identity.go

```diff
@@ -31,6 +31,7 @@ const (
 
 const (
 	AnonymousNamespaceID = NamespaceAnonymous + ":0"
+	GlobalOrgID          = int64(0)
```

Comment:
nit: I'd prefer if we defined all the constants in one place (eg, use the constants from accesscontrol/models.go everywhere instead of reintroducing them).

Reply:
We get a cyclic import if we do so :disappointed: